src/networks.json

```diff
@@ -1330,6 +1330,22 @@
     "start": 1683285,
     "logo": "ipfs://QmVh5XRi7xN4UNw7SNKpEhjF1TsPXJnRyNkB4EuMVTVCB6"
   },
+  "9052": {
+    "key": "9052",
+    "name": "Acrechain Mainnet",
+    "shortName": "Acrechain",
+    "chainId": 9052,
+    "network": "mainnet",
+    "multicall": "0x5B614BB4b2939dc4E5EC1399F16c7E78949764CB",
```

Comment:
Code here https://acrescout.mindheartsoul.org/address/0x5b614bb4b2939dc4e5ec1399f16c7e78949764cb/contracts#address-tabs should be verified and published